### ISSUE TITLE
Problems with Network Mode: Host

### ISSUE THREAD

**Issue by codechimp** [zalando/nakadi#413]
In the current local Nakadi appliance, I experience problems with the network_mode _host_ (I would need to set up my iptables etc to get it work (see also http://www.dasblinkenlichten.com/docker-networking-101-host-mode/)). Actually, you could also design the docker-compose file with _links_:

``` yml
version: '2'
services:

  nakadi:
    build: .
    ports:
     - "8080:8080"
    depends_on:
     - postgres
     - zookeeper
     - kafka
    environment:
      - SPRING_PROFILES_ACTIVE
      - NAKADI_OAUTH2_MODE=OFF
      - NAKADI_FEATURETOGGLE_DEFAULT=true
      - NAKADI_ZOOKEEPER_BROKERS=zookeeper:2181
      - SPRING_DATASOURCE_URL=jdbc:postgresql://postgres:5432/local_nakadi_db
      - SPRING_DATASOURCE_USERNAME=nakadi
      - SPRING_DATASOURCE_PASSWORD=nakadi
    links:
      - zookeeper:zookeeper
      - postgres:postgres

  postgres:
    image: postgres:9.5
    ports:
      - "5432:5432"
    volumes:
      - "./database/nakadi:/docker-entrypoint-initdb.d"
    environment:
      POSTGRES_USER: nakadi
      POSTGRES_PASSWORD: nakadi
      POSTGRES_DB: local_nakadi_db

  zookeeper:
    image: wurstmeister/zookeeper:3.4.6
    ports:
      - "2181:2181"

  kafka:
    image: wurstmeister/kafka:0.9.0.0
    ports:
      - "9092:9092"
    depends_on:
      - zookeeper
    links:
      - zookeeper:zookeeper
    environment:
      KAFKA_ADVERTISED_HOST_NAME: localhost
      KAFKA_ADVERTISED_PORT: 9092
      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
      KAFKA_AUTO_CREATE_TOPICS_ENABLE: 'false'
      KAFKA_DELETE_TOPIC_ENABLE: 'true'
      KAFKA_CREATE_TOPICS: "test-topic:8:1"
      KAFKA_NUM_PARTITIONS: 8
      KAFKA_BROKER_ID: 0
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
```

I will prepare a corresponding Pull Request.

**Comment by PetrGlad** [comment]
Direct links are not necessary with recent docker-compose since all containers in config can share same network and find each other via local DNS.

**Comment by lmontrieux** [comment]
Fixed in #980
